tests/radius/eap_test_peer.c

```diff
@@ -0,0 +1,349 @@
+/*
+ * Example application showing how EAP peer code from wpa_supplicant can be
+ * used as a library.
+ * Copyright (c) 2007, Jouni Malinen <j@w1.fi>
+ *
+ * This software may be distributed under the terms of the BSD license.
```

Comment:
Please add upstream source of this file (e.g. git commit and URL)

Reply:
ok